lms/envs/common.py

```diff
@@ -1633,6 +1637,8 @@ def _make_mako_template_dirs(settings):
 # This should be moved into an XBlock Runtime/Application object
 # once the responsibility of XBlock creation is moved out of modulestore - cpennington
 XBLOCK_MIXINS = (LmsBlockMixin, InheritanceMixin, XModuleMixin, EditInfoMixin)
+if SkillTaggingMixin:
+    XBLOCK_MIXINS += (SkillTaggingMixin,)
```

Comment:
Can we enable it in a way that it can easily be disabled using remote config?

Reply:
Yes. We have[ this ](https://github.com/openedx/edx-platform/blob/5c58eb47bd06c0928b5890705be021754422ef51/xmodule/vertical_block.py#L45)waffle switch